### ISSUE TITLE
Can't Install - Cannot Access File

### ISSUE THREAD

**Issue by ajhix36** [Soreepeong/XivAlexander#212]
So when I try to install XiVAlexander, I get this error message. It happens no matter which type of install I try to do.  Happens even when I run the launcher as administrator first.

![image](https://user-images.githubusercontent.com/15352262/144282498-56ebecde-4f88-4ed1-aaf8-67ad5811adec.png)
![image](https://user-images.githubusercontent.com/15352262/144282443-62add664-b9e6-42ab-84d0-12f467c7d958.png)

**Comment by Soreepeong** [comment]
This error message makes no sense. Can you extract the zip into a separate directory, confirm that there are 2 exe, 2 dll, multiple json files and folders in that extracted directory and then run Loader from that extracted directory?

ffmpeg.exe shouldn't even be there. Did you indeed download the file from Github Releases page?

**Comment by ajhix36** [comment]
Ah, I had those in there from the music upgrade. Ok, I completely deleted the files in there, and downloaded fresh, and now this happened.

![image](https://user-images.githubusercontent.com/15352262/144300526-3fae5d44-5ddf-4daf-826f-ebee7f3c4359.png)

**Comment by ajhix36** [comment]
If I do it from a totally different folder it does work, I'm guessing its because it wants to copy itself TO that folder.

**Comment by Soreepeong** [comment]
Did you extract the files into `%LOCALAPPDATA%\XivAlexander\`? Then this definitely is what's going to happen. What made you extract files into that folder, though? Was there anything misreading on the wiki?

**Comment by ajhix36** [comment]
Yes, I had extracted the new files in there, but not the very first time I installer. The reason is when I had tried to update it while in game, via the check for updates menu, it failed for me, and just left an updates folder, while remaining in the prior version, so I figured I would just replace the files manually. And thus this issue, which now makes sense to me, as it's trying to copy files into that same folder....I would not have expected that behavior necessarily when using the install to game menu in the launcher though, since it makes me target the game folded, I assumed it would just move a file over there.

**Comment by ajhix36** [comment]
![image](https://user-images.githubusercontent.com/15352262/144659012-d908f9d1-ce0d-45c5-bbbc-8a870018e18e.png)

**Comment by Soreepeong** [comment]
This is expected when trying to use updater when XivAlexander is running as installed mode, and updating when game is running while installed is simply impossible. I'll make Check for Updates open Github Release page instead.

**Comment by harmlesskid** [comment]
![aaa](https://user-images.githubusercontent.com/74108839/148635149-6f3caf7d-bf68-4518-a6ee-9c6ea0c12e05.png)
help me aaa, i downloaded several times cuz idk, maybe corrupted files or something, reinstalled ffxiv and still the same error
:(
